### PR TITLE
default    .conf

### PR DESCRIPTION
**Pull request by alexeyneu**

### CODE REVIEW

**Comment by alexeyneu** [comment]
https://travis-ci.org/navcoin/navcoin-core/jobs/540447054#L1711  
this looks like travis failed to mine a block  
https://github.com/zcash/zcash/issues/2929#issuecomment-406673298

**Comment by mxaddict** [comment]
> https://github.com/navcoin/navcoin-core/blob/498e75f2b915bf5d8fccc10c1080ed68b478fceb/src/util.cpp#L719
> 
> 
> waste coz it's closed by end-of-scope

Not sure what you mean by this.

Nut the changes look good, I will test once I get some spare time.

File: src/qt/intro.cpp [hunk 1]
```diff
@@ -226,6 +228,65 @@ void Intro::pickDataDirectory()
         SoftSetArg("-datadir", GUIUtil::qstringToBoostPath(dataDir).string()); // use OS locale for path setting
 }
 
+void Intro::makeDefaultConfF()
+{
+    bool alreadyInConfigFile = false;
```
Comment (aguycalled) [inline src/qt/intro.cpp]: this variable is unused

**Comment by alexeyneu** [comment]
> > https://github.com/navcoin/navcoin-core/blob/498e75f2b915bf5d8fccc10c1080ed68b478fceb/src/util.cpp#L719
> > 
> > waste coz it's closed by end-of-scope
> 
> Not sure what you mean by this.
> 
> But the changes look good, I will test once I get some spare time.  
i mean that it closes as it goes out of scope even without this line.

**Comment by mxaddict** [comment]
Ahh, I see.

**Comment by mxaddict** [comment]
testedACK (Ubuntu 18.04)

Default file created successfully nice work @alexeyneu

**Comment by mxaddict** [comment]
This closed #407

**Comment by aguycalled** [comment]
tested OSX, could you share a Nav address for the bounty payment? @alexeyneu

**Comment by mxaddict** [comment]
OFF TOPIC, @alexeyneu what is your handle on navcoin discord? :+1:

**Comment by alexeyneu** [comment]
NSBWLZUn5cYFsS74LjyyD6kkTX3txfT9pL

**Comment by aguycalled** [comment]
Paid edf746f8aaaa81e7300c75bd38585256ad3c2e13a17455844ccf203da6e7a251

**Pull request merged**